src/taxonomy/export-modal/index.jsx

```diff
@@ -50,12 +51,14 @@ const ExportModal = ({
             <Form.Radio
               key={`export-csv-format-${taxonomyId}`}
               value="csv"
+              labelClassName="export-modal-radio"
```

Comment:
I don't think this class is being defined anywhere.

Reply:
Updated https://github.com/openedx/frontend-app-course-authoring/pull/723/commits/dab9d8a01a0389826e989252b112c3b70869e251